staging/src/k8s.io/apiserver/pkg/storage/tests/cacher_test.go

```diff
@@ -148,7 +149,10 @@ func updatePod(t *testing.T, s storage.Interface, obj, old *example.Pod) *exampl
 func TestGet(t *testing.T) {
 	server, etcdStorage := newEtcdTestStorage(t, etcd3testing.PathPrefix())
 	defer server.Terminate(t)
-	cacher, _ := newTestCacher(etcdStorage, 10)
+	cacher, _, err := newTestCacher(etcdStorage, 10)
+	if err != nil {
+		t.Fatalf("Couldnt' create cacher: %v", err)
```

Comment:
nite: s/Couldnt'/Couldn't/ (for below instances as well)

Reply:
Done.